Common/TableProducer/Converters/converter.cxx

```diff
@@ -0,0 +1,615 @@
+        zdcEnergy.emplace_back(energySectorZPA[ic]);
+        zdcChannelsE.emplace_back(IdZPA1 + ic);
+        zdcEnergy.emplace_back(energySectorZPC[ic]);
+        zdcChannelsE.emplace_back(IdZPC1 + ic);
+      }
+
+      Zdcs_001(bc,
+               zdcEnergy,
+               zdcChannelsE,
+               zdcAmplitudes,
+               zdcTime,
+               zdcChannelsT);
+    }
+  }
+  PROCESS_SWITCH(zdcConverter, processConverter, "Process converter (autoset)", true);
+};
+
+struct mcCollisionConverter {
+  Produces<aod::McCollisions_001> mcCollisions_001;
+  void init(o2::framework::InitContext& initContext)
+  {
+    doprocessConverter.value = isTableRequiredInWorkflow(initContext, "McCollisions_001");
+    if (doprocessConverter) {
+      LOG(info) << "Enabling converter for table McCollisions_001";
+    }
+  }
+  void processConverter(aod::McCollisions_000 const& mcCollisionTable)
+  {
+    for (auto& mcCollision : mcCollisionTable) {
+
+      // Repopulate new table
+      mcCollisions_001(
+        mcCollision.bcId(),
+        mcCollision.generatorsID(),
+        mcCollision.posX(), mcCollision.posY(), mcCollision.posZ(),
+        mcCollision.t(), mcCollision.weight(),
+        mcCollision.impactParameter(),
+        0.0f); // dummy event plane, not available in _000
+    }
+  }
+  PROCESS_SWITCH(mcCollisionConverter, processConverter, "Process converter (autoset)", true);
+};
+
+struct MftTracksConverter {
+  Produces<aod::StoredMFTTracks_001> mftTracks_001;
+  void init(o2::framework::InitContext& initContext)
+  {
+    doprocessConverter.value = isTableRequiredInWorkflow(initContext, "StoredMFTTracks_001");
+    if (doprocessConverter) {
+      LOG(info) << "Enabling converter for table StoredMFTTracks_001";
+    }
+  }
+  void processConverter(aod::MFTTracks_000 const& mftTracks_000)
+  {
+
+    for (const auto& track0 : mftTracks_000) {
+      uint64_t mftClusterSizesAndTrackFlags = 0;
+      int8_t nClusters = track0.nClusters();
+
+      for (int layer = 0; layer < 10; ++layer) {
+        mftClusterSizesAndTrackFlags &= ~(0x3fULL << (layer * 6));
+        mftClusterSizesAndTrackFlags |= (layer < nClusters) ? (1ULL << (layer * 6)) : 0;
+      }
+
+      mftTracks_001(track0.collisionId(),
+                    track0.x(),
+                    track0.y(),
+                    track0.z(),
+                    track0.phi(),
+                    track0.tgl(),
+                    track0.signed1Pt(),
+                    mftClusterSizesAndTrackFlags,
+                    track0.chi2(),
+                    track0.trackTime(),
+                    track0.trackTimeRes());
+    }
+  }
+  PROCESS_SWITCH(MftTracksConverter, processConverter, "Process converter (autoset)", true);
+};
+
+/// Spawn the extended table for MFTTracks001 to avoid the call to the internal spawner and a consequent circular dependency
+struct MFTTracksSpawner {
+  Spawns<aod::MFTTracks_001> mftTracks_001;
+};
+
+struct MultsExtraConverter {
```

Comment:
This doesn't seem to produce something in the central data model? Where is this converter from?